packages/dev/src/examples/misc/treemap/search/index.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState, useCallback } from 'react'
+import { format } from 'd3-format'
+import { VisSingleContainer, VisTooltip, VisTreemap } from '@unovis/react'
+import { Position, Treemap, TreemapNode } from '@unovis/ts'
+import s from './styles.module.css'
+
+export const title = 'Treemap: Search'
+export const subTitle = 'Demo of search by nodes feature'
+
+// Number formatter for population values,
+// using B for billions and M for millions.
+const populationFormatRaw = format('.3s')
+const populationFormat = (value: number): string => populationFormatRaw(value)
+  .replace('G', 'B')
+
+const getTooltipContent = (d: TreemapNode<TreemapExampleDatum>): string => {
+  const isLeafNode = d.data.datum !== undefined
+  return isLeafNode ? `
+    <div class="${s.tooltipTitle}">${d.data.datum.name}</div>
```

Comment:
I see some type errors in this example that would be great to resolve.
<img width="1074" height="270" alt="Image" src="https://github.com/user-attachments/assets/340d9588-871c-48d1-97b7-f99d2233070e" />

Also, managing color persistence is currently outside the library's responsibility, but it would be nice to show in this example how to have persistent colors. Currently when you search, the color of the tiles changes

https://github.com/user-attachments/assets/1a10071b-512a-4a86-96b5-acc694dcc43d